help/destinations/catalog/advertising/criteo.md

```diff
@@ -92,6 +95,7 @@ After authenticating to the destination, please fill in the following connection
 | Description | A description to help you identify this destination in the future. | No |
 | API Version | Criteo API Version. Please select Preview. | Yes |
 | Advertiser ID | Criteo Advertiser ID of your organization. Please contact your Criteo account manager to obtain this information. | Yes |
+| Criteo gum caller ID | Gum Caller Id of your organization. Please reach out to your Criteo account team for the appropriate gumCallerId or to get more information on this GUM sync, if needed. | No |
```

Comment:
The GUM caller ID is marked as optional in the docs, while the submitted destination config has it marked as required. Is it required or optional?